src/routes/landing/components/Heading.js

```diff
@@ -4,14 +4,19 @@ import LocationBig from '../assets/LocationBig.svg'
 import style from './Heading.css'
 import { translate } from 'react-i18next'
 import Caption from 'modules/common/components/Caption'
+import PropTypes from 'prop-types'
 
 class Heading extends React.Component {
+  static propTypes = {
+    t: PropTypes.func.isRequired
+  }
+
   render () {
     const {t} = this.props
```

Comment:
you could inline this

Reply:
Inline how? (Give always the example in such cases :P)

Reply:
this.props.t('where') instead of t('where')